specification/cognitiveservices/data-plane/ImageSearch/readme.nodejs.md

```diff
@@ -6,7 +6,7 @@ Please also specify `--node-sdks-folder=<path to root folder of your azure-sdk-f
 ``` yaml $(nodejs)
 nodejs:
   package-name: azure-cognitiveservices-imagesearch
-  package-version: 1.0.0-preview
+  package-version: 1.0.0
```

Comment:
Just curious, are these parts of Cognitive service live separately? So there is a package for VideoSearch and then another one for VisualSearch?

Reply:
Yeah, we publish a different npm package for each one.